### DISCUSSION TITLE
Backtrack temp file name to the actual command

### DISCUSSION THREAD

**Discussion by Nabarun** [ImageMagick/ImageMagick#5185] [Development] [answered]
I am seeing magick files getting created in the tmp folder. Is there a way to log the command for the same, so that we can map the command which got executed and the temp magick file which got created. It will help investigate which command caused the tmp file creation in case it doesn't gets auto deleted.

**Comment by urban-warrior** [comment, marked as answer]
Add `-debug Resource` to your command-line.

**Reply by Nabarun** [reply]
Thanks let me try
